prow/getting_started_deploy.md

```diff
@@ -15,6 +15,8 @@ You need three things:
 2) a [personal access token][1] for the github bot user you want to act as prow
 
    - Prow uploads this token into a secret in your cluster
+   - The token must be stored in a text file
```

Comment:
As opposed to what? What constitutes a text file? Maybe we should write

    The token should be saved in plain text

?

Reply:
Sure, sounds good, I'll update it, thx for the review